tests/tmt/tests/jats/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -eux
+
+TESTS_RESULT=test_results
+JAT_LOG="/var/tmp/jat/last/log.yaml"
+JAT_RELICS="/var/tmp/jat/last/relics/xcase-results.yaml"
+
+suite=${TESTSUITE:-leapp}
+testname=${TESTNAME:-cli}
+
+jats_test="/usr/share/jats/suite/$suite/$testname"
+test_dir_name=$(echo "$jats_test" | sed -e 's/\//_/g')
+echo "Processing $jats_test.."
+# perform cleanup just in case previous test failed hard
+if [ -d /var/tmp/jat ]; then
+    rm -r /var/tmp/jat
+fi
```

Comment:
OTOH, if the previous test failed hard (timed out?) the system is likely in an unknown state, hence the following tests can't be trusted.

Not sure if there's a nice solution, though.  (The *right* one is to parallelize but AFAIK we can't do that now,)  At the very least whoever is reading the result needs to be extremely careful to consider the state the system test was left in.

Reply:
We can easily parallilize tests at the launcher level - whatever or whoever is launching the tests can make it run in parallel with adding --filter 'tier:somespecifictier' .